src/components/blocks/search-form/_search-form.scss

```diff
@@ -482,13 +568,55 @@
       margin: 0 20px 10px;
     }
   }
+  
+  &__advanced-search-wrapper {
+    color: base.sitecolors-siteColor("vam-black");
+    display: block;
+    flex: 0 0 100%;
+    height: auto;
+    margin: 0 0 8px;
+    opacity: 1;
+    visibility: visible;
+    width: auto;
+    position: relative;
+
+    &::after {
+      content: "⌄";
+      font-size: 20px;
+      font-weight: 100;
+      height: 10px;
+      pointer-events: none;
+      position: absolute;
+      right: 18px;
+      top: 12%;
+      transform: scaleX(1.5);
+      width: 10px;
+      
+      @include mixins.breakpoints-bpMinSmall {
+        top: 12%;
+      }
+
+      @include mixins.breakpoints-bpMinMedium {
+        top: 24%;
+      }
+    }
+
+    @include mixins.breakpoints-bpMinXSmall {
+      margin-left: 0;
+    }
+
+    @include mixins.breakpoints-bpMinSmall {
+      flex: 0 0 auto;
+      margin-bottom: 0;
+    }
```

Comment:
Given the conflict between design and browser behaviour where width of the select is concerned, I suggest  we compromise with a fixed flex-basis of 10em (i.e. `flex: 0 0 auto`), and set `text-overflow: ellipsis;` on the `__filter-select` (can do that in the standard rule for `&__filter-select`, since it can apply everywhere).